services/src/app.js

```diff
@@ -49,10 +50,8 @@ logger.level = process.env.LOG_LEVEL || 'warn';
  */
 if (process.env.SENTRY_DSN) {
   logger.info('Configuring Sentry for backend errors..');
-  const Raven = require('raven');
   Raven.config(process.env.SENTRY_DSN).install();
   app.use(Raven.requestHandler());
-  app.use(Raven.errorHandler());
   logger.info('..Sentry middleware installed');
 }
 
```

Comment:
Is there a reason the rest of the logic wouldn't also move to the end of app startup?  It is better for code readability if all the configuration for Sentry is all together.

Reply:
Yeah, the `requestHandler` has  to be the first middleware :man_facepalming: 

It's derpy

Reply:
Super derpy